neuraxle/base.py

```diff
@@ -658,25 +668,15 @@ def teardown(self) -> 'BaseStep':
 
     def set_train(self, is_train: bool=True):
         """
+        This method overrides the method of BaseStep to also consider the wrapped step as well as self.
```

Comment:
Sorry, it's hard to review in the Github UI because the class is long. Is this a method of Base Step or of MetaStepMixin? If we're in Base Step right now, the comment was good as it was before with an example on how to split like this: 

```
            def transform(self, data_inputs):	
                if self.is_train:	
                    self.transform_train_(data_inputs)	
                else:	
                    self.transform_test_(data_inputs)	
            def fit_transform(self, data_inputs, expected_outputs):	
                if self.is_train:	
                    self.fit_transform_train_(data_inputs, expected_outputs)	
                else:	
                    self.fit_transform_test_(data_inputs, expected_outputs)	
```

Sorry for the mistake if I mixed this method with the one of MetaStepMixin while reviewing.